modular_splurt/code/modules/client/preferences.dm

```diff
@@ -16,113 +20,497 @@
 		/datum/reagent/consumable/ethanol/cogchamp,
+
+		// Heals body part and brute damage for some mobs
 		/datum/reagent/consumable/ethanol/pinotmort,
+
+		// Heals body part and brute damage for security
 		/datum/reagent/consumable/ethanol/quadruple_sec,
+
+		// Heals body part, brute, suffocation, burn, and toxin damage for security
 		/datum/reagent/consumable/ethanol/quintuple_sec,
+
+		// Heals brute, burn, toxin, suffocation, and stamina damage
 		/datum/reagent/consumable/ethanol/bastion_bourbon,
+
+		// Grants nutrition
 		/datum/reagent/consumable/ethanol/squirt_cider,
+
+		// Grants nutrition
 		/datum/reagent/consumable/ethanol/sugar_rush,
-		/datum/reagent/consumable/ethanol/crevice_spike,
+
+		// Grants soothed throat effect and increases temperature
 		/datum/reagent/consumable/ethanol/peppermint_patty,
+
+		// Removes mighty shield reagent
 		/datum/reagent/consumable/ethanol/alexander,
+
+		// Heals brute and burn damage for sleeping users
 		/datum/reagent/consumable/ethanol/between_the_sheets,
+
+		// Removes nutrition and causes toxin damage
 		/datum/reagent/consumable/ethanol/fernet,
+
+		// Removes nutrition and causes toxin damage
 		/datum/reagent/consumable/ethanol/fernet_cola,
+
+		// Removes nutrition and clears overeating duration
 		/datum/reagent/consumable/ethanol/fanciulli,
+
+		// Reduces body temperature
 		/datum/reagent/consumable/ethanol/branca_menta,
+
+		// Heals body part damage for mimes
 		/datum/reagent/consumable/ethanol/blank_paper,
+
+		// Heals body part, suffocation, and toxin damage for wizards
 		/datum/reagent/consumable/ethanol/wizz_fizz,
+
+		// Causes toxin damage to insects
 		/datum/reagent/consumable/ethanol/bug_spray,
+
+		// Reduces stamina
 		/datum/reagent/consumable/ethanol/turbo,
+
+		// Increases age, changes hair color, causes nearsightedness, causes a beard
 		/datum/reagent/consumable/ethanol/old_timer,
+
+		// Heals burn damage, removes jittering, and removes stuttering for Chaplain
 		/datum/reagent/consumable/ethanol/trappist,
+
+		// Teleports the user
 		/datum/reagent/consumable/ethanol/blazaam,
+
+		// Increases temperature, and can cause ignition
 		/datum/reagent/consumable/ethanol/mauna_loa,
+
+		// Heals body part, brute, suffocation, fire, foxin, and radiation for the Captain
 		/datum/reagent/consumable/ethanol/commander_and_chief,
+
+		// Displays a chat message
+		// /datum/reagent/consumable/ethanol/gunfire,
+
+		// Increases temperature
 		/datum/reagent/consumable/ethanol/hellfire,
-		//Drink reagents
+
+		// Causes drugging and stamina loss
+		/datum/reagent/consumable/ethanol/hotlime_miami,
+
+		// Plays an explosion sound effect
+		/datum/reagent/consumable/ethanol/b52,
+
+		// Causes brute damage
+		/datum/reagent/consumable/ethanol/crevice_spike,
+
+		//
+		// SPLURT effect drinks
+		//
+
+		// Causes clothing loss
+		/datum/reagent/consumable/ethanol/panty_dropper,
+
+		// Causes brain damage
+		/datum/reagent/consumable/ethanol/lean,
```

Comment:
these may have some effects but it'd be good to keep them given they're more or less just memes of the server